<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="VerticalLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="HorizontalLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Razor">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/sourse/razor.png</normaloff>:/new/prefix1/sourse/razor.png</iconset>
   </property>
   <property name="text">
    <string>视频切分</string>
   </property>
  </action>
  <action name="StartCut">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/sourse/StartCut.bmp</normaloff>:/new/prefix1/sourse/StartCut.bmp</iconset>
   </property>
   <property name="text">
    <string>开始截取</string>
   </property>
  </action>
  <action name="EndCut">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/new/prefix1/sourse/EndCut.bmp</normaloff>:/new/prefix1/sourse/EndCut.bmp</iconset>
   </property>
   <property name="text">
    <string>结束截取</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
